doc/changes/changes_4.0.4.md

```diff
@@ -12,7 +12,8 @@ Code name:
 ## Refactoring
 
 * #263: Removed SybaseMetadataReader class as it was not used by the dialect.
-* #381: Migrated from version.sh to artifact-reference-checker-maven-plugin
```

Comment:
Add artifact-reference-checker-maven-plugin  to plugin list dependency in the readme

Reply:
It's already there, I think you added it :D

Reply:
ok, then.